TraceScripts/README.md

```diff
@@ -30,3 +30,74 @@ xbtrace /x:<xboxip> start advancedmp
 ```
 xbtrace /x:<xboxip> stop
 ```
+
+## iOS - macOS
+
+The PlayFab Multiplayer C++ library for iOS and macOS includes logging capabilities with a configurable verbosity level. Logging configuration is defined in `PlayFabMultiplayerLogger.json` file which can be deployed as an asset along with the application.
+
+When this file is detected by the application at runtime it will use it to enable logging as configured. The following verbosity levels are currently supported:
+1. `VERBOSE` - everything
+2. `INFO` - less than everything, only important messages and errors
+3. `ERROR` - only errors
+
+The numeric values of the "logAreaFlags_ApiInOut", "logAreaFlags_FnInOut", and "logAreaFlags_Log" are internal and users should not need to alter them.
+
+### iOS
+
+#### Example configuration file
+
+```json
```

Comment:
I considered also creating a separate file: PlayFabMultiplayerLogger.json and add the contents there, but given that the files are not identical for iOS, macOS I would have to either:
1. Create an iOS and a macOS directory and place the file under the new directories.
2. Rename them as: PlayFabMultiplayerLogger-iOS.json and PlayFabMultiplayerLogger-macOS.json (names are arbitrary) but then would have to document that the file has to be renamed to the original name.

There might be other ways also, please let me know your thoughts on that.

Reply:
Of the 3 choices (what's currently in the PR vs option 1 vs option 2), I like either what's currently in the PR or option1

it seems easy to forget to rename the files or skim over that when reading the instructions

and it's just as easy to link the files from this readme whether they are alongside this README or nested in a subfolder

I like what's currently in the PR since it's nice that the config files are just there front and center BUT it occurs to me that basically the same logging instructions exist for Android and Linux also so if we ever add those instructions it'll start to bloat this readme quite a bit.

so maybe option 1 is best and then just provide one example configuration? or maybe just use the example configuration files section as a place to list the different links to each platform's config file

e.g.

```
#### Example configuration files

* [iOS](/ios/PlayFabMultiplayerLogger.json)
* [macOS](/macos/PlayFabMultiplayerLogger.json)
```

Reply:
I am leaning towards option 1 along with: _"or maybe just use the example configuration files section as a place to list the different links to each platform's config file_".

It is highly likely that we will add instructions for Android and Linux soon, and in that case this README would grow a lot in size.